c10/test/util/complex_test_common.h

```diff
@@ -389,6 +389,32 @@ MAYBE_GLOBAL void test_arithmetic() {
   test_arithmetic_<double>();
 }
 
+template<typename T, typename int_t>
+static void test_binary_ops_for_int_type_(T real, T img, int_t num) {
+  c10::complex<T> c(real, img);
+  ASSERT_EQ(c + num, c10::complex<T>(real + num, img + num));
```

Comment:
Shouldn't the result be `c10::complex<T>(real + num, img)`

Reply:
You are right, fixed.